readme.md

```diff
@@ -132,6 +132,23 @@ slugify('foo@unicorn', {
 //=> 'foo-at-unicorn'
 ```
 
+##### preserveLeadingUnderscore
+
+Type: `boolean`\
+Default: `false`
+
+If your string starts with an underscore, it will be preserved in the slugified string.
```

Comment:
You need to keep the readme in sync with the index.d.ts docs.

Reply:
I think it would also be useful to include a use-case for this option, as mentioned by the original issue.

Reply:
I have just added an example as a hidden_filename to give this idea

Reply:
Sorry, I should have been clearer, by use-case, I meant a written use-case, not a code example. I don't think `_hidden-filename` clarifies much.

Reply:
Right. and if I just update the description to:
"If your string starts with an underscore, it will be preserved in the slugified string. Used in filenames to represent a hidden path."

Reply:
I still don't think that's very clear.

You could almost just copy-paste the comment from the original issue:

> Sometimes leading underscores are intentional, for example, filenames representing hidden paths on a website.

Reply:
ok, perfect! done that!